auto_round/mllm/autoround_mllm.py

```diff
@@ -168,12 +168,15 @@ def __init__(
                 truncation = False
                 seqlen = 512 if seqlen is None else seqlen
                 if batch_size != 1:
-                    logger.warning(f"batch_size({batch_size}) is not supported for dataset {dataset}, rest to 1")
+                    logger.warning(
+                        f"batch_size({batch_size}) is not supported for dataset {dataset}, reset to 1" 
+                        f" and gradient_accumulate_steps reset to {batch_size * gradient_accumulate_steps}")
                     gradient_accumulate_steps = batch_size * gradient_accumulate_steps
                     batch_size = 1
         if quant_nontext_module and batch_size != 1:
-            logger.warning(f"batch_size({batch_size}) cannot be used for calibrating non-text modules,"
-                           "reset to 1")
+            logger.warning(
+                f"batch_size({batch_size}) cannot be used for calibrating non-text modules, reset to 1"
+                f" and gradient_accumulate_steps reset to {batch_size * gradient_accumulate_steps}")
```

Comment:
better follow this style, 
reset bs to 1 and grad to xxx as bs=8 cannnot be used,